src/Pages/AddItem.js

```diff
@@ -72,3 +79,19 @@ export default function AddItem({ token }) {
     </div>
   );
 }
+const validateNewListItem = (listItem, list) => {
+  const punctuation = /[!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~]/g;
+
+  const filteredList = list.docs.filter((filteredItem) => {
```

Comment:
We don't need necessary need to update this for this MR, but here's a few thoughts about choosing the "right" method:

There's 100 ways to solve any given problem in JS. I tend to lean towards the options that make my code the most readable as possible. I try to make my variables read like english and, when possible, have my methods extend that ideology. There's a couple things we can do here to improve this code with that in mind:

1). We could start by using `.find` instead of `.filter`. I "find" that `.find` is a bit clearer with our intentions, since we only care about finding a single match. If for what ever reason we wake up one day and decide we want to support a user adding up to three duplicates, filter would make perfect sense. But since we're only interesting in knowing if there's a single match, `find` makes a bit more sense to me. It also has the added benefit of the smallest performance increase. 'Filter' has to loop through the entire array, 'find' only has to loop through the array until it finds a match. 

2. If we decided to update the method we used here, we the rest of our code would be a touch more readable too. We could update this variable name from `filteredList` to `matchingItem`, and return `matchingItem` instead of checking for a length of array. We would probably take this one small step forward, though. We wouldn't want to return just `matchingItem` because that would either be the item we matched on or `undefined`, which isn't a boolean. Here is where we could leverage one of my favorite JavaScript tricks: the double bang! It transforms "truthy" and "falsy" values into either `true` or `false`. [Here's a short write up on that](https://dev.to/sanchithasr/what-is-the-double-bang-operator-in-javascript-4i3h)

Reply:
Thank you for the insight! I agree, .find would be great to use in this case. For the variable name, would `duplicateItem` work for readability?

Reply:
I think that's perfect 👍